sky/clouds/aws.py

```diff
@@ -466,6 +467,14 @@ def make_deploy_resources_variables(
         image_id = self._get_image_id(image_id_to_use, region_name,
                                       resources.instance_type)
 
+        ssh_user = skypilot_config.get_effective_region_config(
+            cloud='aws',
+            region=region_name,
+            keys=('ssh_user',),
+            default_value=None)
+        if not ssh_user:
+            ssh_user = DEFAULT_SSH_USER
```

Comment:
```suggestion
            default_value=DEFAULT_SSH_USER)
```

Can we simplify this?

Reply:
Updated